docs/reference/objects/product/variant/index.md

```diff
@@ -105,6 +112,8 @@ number
 {: .label .fs-1 }
 
 The initial stock for the variant, if the variant has unlimited inventory this will return `nil`.
+- When accessed through an [experience date]({% link docs/reference/objects/product/experience_date.md %}), it will return the initial stock for the variant on the given date.
+- When accessed independently or through a product, it will return the default initial stock for the variant, managed in **Experience > Variants**.
```

Comment:
Should this now be updated based on this thread? https://easolhq.slack.com/archives/C03AJ5F6L1L/p1689348944793599